common/lib/xmodule/xmodule/library_tools.py

```diff
@@ -96,6 +96,17 @@ def _filter_child(self, usage_key, capa_type):
         assert isinstance(descriptor, CapaDescriptor)
         return capa_type in descriptor.problem_types
 
+    def can_use_library_content(self, course_id):
+        """
+        Determines whether a modulestore holding a course_id supports libraries.
+        """
+        store = self.store
+        # Skip MixedModuleStore abstraction.
+        if hasattr(store, '_get_modulestore_for_courseid'):
```

Comment:
@Kelketek Looks like you rebased this and missed changing this `hasattr` string to match the new method name two lines down.